.vale/styles/style_guide/Indexing.yml

```diff
@@ -0,0 +1,10 @@
+---
+# Error: style_guide.indexing.yml
+
+# Most dictionaries still list "indexation" as a term with a very specific meaning in economy so we should use “indexing”
+extends: substitution
+message: "Use '%s' instead of '%s'."
+level: warning
+swap:
+  indexation: indexing
+  
```

Comment:
```suggestion

```

Reply:
All the rules have a blank line at the end, so I guess I'll keep it 🤷🏻‍♀️

Reply:
yeah, the blank line is good, but the problem that github was complaining about was that the line had two spaces, so it wasn't technically blank.

Reply:
Got it, I'll fix this in another PR